Documentation/ApiOverview/MountPoints/Index.rst

```diff
@@ -1,180 +1,75 @@
-.. include:: /Includes.txt
-.. index:: Mount points
```

Comment:
Just for my understanding, what is this for?

Reply:
① `.. include:: /Includes.txt` We once adopted the convention (from the NEOS team) to always have that include at the beginning. Adding something to the Include.rst.txt make that available everywhere in the manual. For example, you could define frequently used hyperlinks there.

② NEW: I've added the 'Index' ("Stichwortverzeichnis") page to the manual. https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/genindex.html This means, we can start using Index entries with the indexing directive: `.. index:: …`. Once there are more entries this will look similar to https://docs.typo3.org/m/typo3/guide-contributionworkflow/master/en-us/genindex.html

* https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-index
* https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#role-index
* https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#index-generating-markup

Reply:
I was asking about the index because @susannemoog removed it again. That's a really cool feature!

Reply:
Yes. 

We should go for Index pages in all manuals. Especially, as some other directives _automatically_ created entries there, for example the `.. confval:` directive creates 'configuration value' entries automatically. If we used `.. confval::` in our TypoScript reference there would automatically be an index of TypoScript objects! Demo example: https://docs.typo3.org/m/typo3/demo-t3SphinxThemeRtd/master/en-us/genindex.html#C
The Index page is already part of the manual. Only: It is still empty. If Susanne had kept the line, there would be at least that one single entry. I intended to have that as a demonstration of how to do it and as a reminder, that someone may have fun to define a lot of index entries in one go. Well, once somebody enjoys doing that, the index entry 'Mount points' can still be re-introduced. May be - you? - or somebody else has some fun adding index entries? Should be announced in the #documentation-team channel. I just added you there.

Reply:
That was accidentally removed by the merge, thanks for adding it again. I think this PR is done now from my side and we could merge it this way. We can of course keep improving the docs after that - I just think it would be good to have the base in :)